### PR TITLE
fix: Update changelog workflow

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

**Review by ad1992 — APPROVED** [review]

File: .github/workflows/changelog-check.yml [hunk 1]
```diff
@@ -2,9 +2,11 @@ name: Changelog in sync for packages
 
 on:
   push:
-    branches:
-      - master
   pull_request:
+    branches-ignore:
+      - l10n_master
+      - dependabot/*
+      - chenagelog # just testing
```
Comment (dwelle) [inline .github/workflows/changelog-check.yml:9]: ?

Comment (lipis) [reply .github/workflows/changelog-check.yml:9]: I was testing to ignore this branch.. but it doesn't seem that it's skipping it.

Comment (dwelle) [reply .github/workflows/changelog-check.yml:9]: maybe because it's misspelled

Comment (lipis) [reply .github/workflows/changelog-check.yml:9]: Well it's the same as the current branch name, as I copy pasted it.. 😄

Comment (lipis) [reply .github/workflows/changelog-check.yml:9]: My guess would be that that shouldn't run at all if it's ignored..
Screenshot 2020-12-14 at 3 17 05 PM

**Comment by lipis** [comment]
Let's continue in #2480

**Pull request closed without merging**